sdk/core/azure-core/src/exception.cpp

```diff
@@ -3,10 +3,12 @@
 
 #include "azure/core/exception.hpp"
 #include "azure/core/http/http.hpp"
-
+#include <azure/core/http/policies/policy.hpp>
+#include <azure/core/internal/json/json.hpp>
```

Comment:
The files are local so they should be using "" not <>.  We use "" for files within the package and <> for files coming from other packages

Reply:
done